src/dvclive/live.py

```diff
@@ -95,8 +97,8 @@ def __init__(
         self._report_notebook = None
         self._init_report()
 
-        self._baseline_rev: Optional[str] = None
-        self._exp_name: Optional[str] = exp_name
+        self._baseline_rev: str = os.getenv(env.DVC_EXP_BASELINE_REV, NULL_SHA)
```

Comment:
Q: why using a special SHA? can we just keep it None - that would be more explicit I think

Reply:
Just going for the simplest solution rather than one that requires updates in dvc-studio-client (see https://github.com/iterative/dvc-studio-client/pull/72) and studio. Didn't feel like there is justification to go for the more complex solution.

Reply:
yep, my concern that we are getting some tech debt here potentially, and then it takes 2x+ time to change it to something. I'm fine to go with this, just want to quickly double check - `dvc-studio-client ` change doesn't look complicated (?), on the Studio side we also can do whatever we want initially (e.g. convert to 40x0 in API if needed - at least we would isolate it to the Studio side vs open source client library?)

Reply:
I will leave it up to you and @mattseddon. Not a strong preference from me. I would rather not need the baseline rev in dvclive at all but don't see an easy way to drop it completely.

Reply:
My $0.02 is that the way `dvc-studio-client`/`dvc`/`dvclive`/`studio` are setup for dev on live experiments at the moment is a complete PITA to work with. Given that `baseline_sha` is required/non-nullable (IIUC) by the model in Studio AND `dvc-studio-client` I would say that we do this for now.

```
class Exp(SoftDeleteModel, LifecycleModel):
    repo_url = models.URLField()
    baseline_sha = models.CharField(max_length=40)
```

```
BASE_SCHEMA = Schema(
    {
        Required("type"): choices("start", "done", "data"),  # No "interrupt" for now
        Required(
            "repo_url",
        ): str,  # TODO: use some url validator, voluptuous.Url is too strict
        Required("baseline_sha"): Sha,
```

I think we should also move `dvc-studio-client` into `dvc` which is (maybe) what was being suggested [here](https://github.com/iterative/dvc/pull/10284#issuecomment-1931266683).

Reply:
It took me awhile to understand your comment here @mattseddon. To confirm:
1. You prefer we keep this PR as is with the `NULL_SHA` value?
2. The code blocks you pasted are the existing snippets from `studio` and `dvc-studio-client` that show that `baseline_sha` is required in both of those projects?
3. You want to move `dvc-studio-client` into `dvc`, but it's not a blocker for this PR?

Reply:
1. yes
2. yes
3. yes
👍🏻

Reply:
Note to self: don't try to resolve the baseline information for this sha.